homework/index.js

```diff
@@ -28,14 +28,75 @@
     });
     return elem;
   }
+  // convert date time
+  function convertDataTime(dateTimeTxt) {
```

Comment:
What is the added value of the comment preceding this function? It just repeats what the function name already indicates (although there is a typo in the function name: it is not `data` but `date`).

Reply:
Dear @remarcmij  firts of all, thank you so much for the valuable feedbacks.

actually you are right there is no added value for a comment here, function name is clesar enough to tell about the code.  I will be more careful when commenting the codes.